src/UITests/GettingStartedPage.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using Axe.Windows.Automation;
+
+namespace UITests
+{
+    [TestClass]
+    public class GettingStartedPage : AIWinSession
+    {
+        [TestMethod]
+        public void VerifyGettingStartedTitle()
+        {
+            Assert.AreEqual(session.Title, "Accessibility Insights for Windows - ");
```

Comment:
Actual and Expected parameters are flipped.

Reply:
Nope. Expected is first, followed by actual [https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.testtools.unittesting.assert.areequal?view=mstest-net-1.2.0#Microsoft_VisualStudio_TestTools_UnitTesting_Assert_AreEqual_System_Object_System_Object](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.testtools.unittesting.assert.areequal?view=mstest-net-1.2.0#Microsoft_VisualStudio_TestTools_UnitTesting_Assert_AreEqual_System_Object_System_Object)_

---
In reply to: [286112657](https://github.com/microsoft/accessibility-insights-windows/pull/362#discussion_r286112657) [](ancestors = 286112657)

Reply:
Karan fixed it in the most recent commit